lib/gui/components/safe-webview.js

```diff
@@ -198,26 +199,29 @@ class SafeWebview extends react.PureComponent {
    * @param {Event} event - event object
    *
    * @example
+   *
+   * // In the webview
    * console.log({
-   *   type: 0,
-   *   message: 'Hello, World!'
+   *   command: 'error',
+   *   data: 'Good night!'
    * });
+   *
+   * // or with robot
+   * robot.printError('Good night!');
    */
   static consoleMessage(event) {
-    const LOG_LEVEL = 0;
-    const WARN_LEVEL = 1;
-    const ERROR_LEVEL = 2;
+    const {
+      command,
+      data
+    } = _.attempt(robot.parseMessage, event.message);
```

Comment:
It's up to @jviotti which style he prefers, but why 'unpack' the object here, instead of just checking e.g. `parsedMessage.command` instead of `command` later on?